front/app/containers/IdeasShow/components/RightColumnDesktop/index.tsx

```diff
@@ -41,6 +42,9 @@ const RightColumnDesktop = ({
 }: Props) => {
   const { data: phases } = usePhases(projectId);
   const { data: idea } = useIdeaById(ideaId);
+  const followEnabled = useFeatureFlag({
+    name: 'follow',
+  });
```

Comment:
The issue was that the box was being rendered even when there were no interactions a user could take (I.e. The idea was not in a current participation context, commenting was not enabled, and follow was not enabled).